src/transformers/modeling_utils.py

```diff
@@ -2809,13 +2801,10 @@ def from_pretrained(
             quantization_method_from_config = config.quantization_config.get(
                 "quant_method", QuantizationMethod.BITS_AND_BYTES
             )
-        if quantization_method_from_config is not None and quantization_method_from_args is not None:
-            if quantization_method_from_config != quantization_method_from_args:
-                raise ValueError(
-                    f"The model is already quantized with {quantization_method_from_config}. "
-                    f"You can't quantize it again with {quantization_method_from_args}"
-                )
```

Comment:
Please keep [this](https://github.com/huggingface/transformers/pull/27500). This is in case someone tries to combine different quantization scheme together (e.g bitsandbytes + gptq)